documentation/modules/ROOT/nav.adoc

```diff
@@ -26,12 +26,17 @@
 ** xref:04-quarkus-extensions.adoc#qext-intro[What are Extensions ?]
 ** xref:04-quarkus-extensions.adoc#qext-list-extensions[List Extensions]
 ** xref:04-quarkus-extensions.adoc#qext-health-checks[Health Checks]
-** xref:04-quarkus-extensions.adoc#qext-add-extensions[Add Extensions]
-** xref:04-quarkus-extensions.adoc#qext-rebuild-container-images[Rebuild Container Images]
-** xref:04-quarkus-extensions.adoc#qext-check-health-endpoint[Check Health Endpoint]
-** xref:04-quarkus-extensions.adoc#qext-call-health-ep[Invoke Health Endpoint]
-** xref:04-quarkus-extensions.adoc#qext-add-probes-to-k8s[Add Health Probes to Kubernetes Application]
-** xref:04-quarkus-extensions.adoc#qext-invoke-k8s-svc[Invoke Service]
+*** xref:04-quarkus-extensions.adoc#qext-add-extensions[Add Extension]
```

Comment:
I have shifted the indentation level because I am planning to add more extensions in this section.